src/api/app/services/trigger_controller_service/scm_extractor.rb

```diff
@@ -43,7 +43,8 @@ def github_extractor_payload
                          # We need this for Workflow::Step#branch_request_content_github
                          source_repository_full_name: @payload.dig(:repository, :full_name),
                          # We need this for SCMStatusReporter#call
-                         target_repository_full_name: @payload.dig(:repository, :full_name) })
+                         target_repository_full_name: @payload.dig(:repository, :full_name),
+                         ref: @payload.dig(:ref) })
```

Comment:
We had to add the ref field to the payload, since its used in the validator to check if its a push event, this got overlooked in a previous PR

Reply:
Its needed here https://github.com/openSUSE/open-build-service/blob/934aa35442f7789ea1ce761f07301f49b36b68f6/src/api/app/validators/scm_webhook_event_validator.rb#L52